aloscene/scene_flow.py

```diff
@@ -0,0 +1,187 @@
+from aloception.aloscene import Depth, CameraIntrinsic, Mask, Flow
+import aloception.aloscene as aloscene
+from typing import Union
+import numpy as np
+import torch
+import torch.nn.functional as F
+
+
+def load_scene_flow(path: str) -> torch.Tensor:
+    with open(path, "rb") as file:
+        data = np.load(file)
+        return torch.from_numpy(data)
```

Comment:
No need to convert to tensor

Reply:
Does it mean that x can also be a numpy array if you don't convert ?

Reply:
As long as your output is used to build an AugmentedTensor, you can just `return data`.